src-tauri/Cargo.toml

```diff
@@ -27,7 +27,7 @@ base64 = "0.22.0"
 bonsaidb = { version = "0.5.0", features = ["local", "async"] }
 dirs = "5.0.1"
 futures = "0.3.30"
-home-config = { version = "0.6.0", features = ["toml"] }
+directories = "5.0.1"
```

Comment:
This should probably be changed in multiple ways:

- instead of `directories`, you can either re-use `dirs`, or better, Tauri's [`app_config_dir`](https://docs.rs/tauri/2.0.0-beta.11/tauri/path/struct.PathResolver.html#method.app_config_dir)

Reply:
`dirs` is a lower-level API and not useful.

Tauri already provides a `directories` API. But then initialization has to be deferred until `setup()` when an `AppHandle` is available.